scripts/kind-e2e/e2e.sh

```diff
@@ -122,6 +122,37 @@ function setup_broker() {
     SUBMARINER_BROKER_TOKEN=$(kubectl --context=cluster1 -n ${SUBMARINER_BROKER_NS} get secrets -o jsonpath="{.items[?(@.metadata.annotations['kubernetes\.io/service-account\.name']=='${SUBMARINER_BROKER_NS}-client')].data.token}"|base64 --decode)
 }
 
+# Only install Submariner on broker cluster with 2 worker nodes but do not tag any worker nodes with gateway label
+function install_subm_in_cluster1() {
+    if kubectl --context=cluster1 wait --for=condition=Ready pods -l app=submariner-engine -n submariner --timeout=60s > /dev/null 2>&1; then
+            echo Submariner already installed, skipping submariner helm installation...
+            update_subm_pods cluster1
+        else
+            echo Installing submariner on cluster1...
+            helm --kube-context cluster1 install submariner-latest/submariner \
+            --name submariner \
+            --namespace submariner \
+            --set ipsec.psk="${SUBMARINER_PSK}" \
+            --set broker.server="${SUBMARINER_BROKER_URL}" \
+            --set broker.token="${SUBMARINER_BROKER_TOKEN}" \
+            --set broker.namespace="${SUBMARINER_BROKER_NS}" \
+            --set broker.ca="${SUBMARINER_BROKER_CA}" \
+            --set submariner.clusterId="cluster1" \
+            --set submariner.clusterCidr="10.244.0.0/16" \
+            --set submariner.serviceCidr="100.94.0.0/16" \
+            --set submariner.natEnabled="false" \
+            --set routeAgent.image.repository="submariner-route-agent" \
+            --set routeAgent.image.tag="local" \
+            --set routeAgent.image.pullPolicy="IfNotPresent" \
+            --set engine.image.repository="submariner" \
+            --set engine.image.tag="local" \
+            --set engine.image.pullPolicy="IfNotPresent"
```

Comment:
We need to also `--set crd.create="false"`.

Reply:
DOne